xknx/devices/climate.py

```diff
@@ -15,6 +16,8 @@
 from .climate_mode import ClimateMode
 from .device import Device
 
+logger = logging.getLogger("xknx_log")
```

Comment:
Shall we use an extra device-logger?

Reply:
I thought about this too, yes sure. I also saw that the other 3 loggers are only really barely used.